lib/src/DTO/NotificationAndroid.ts

```diff
@@ -6,12 +6,17 @@ export class NotificationAndroid extends Notification {
     this.identifier = this.payload["google.message_id"];
   }
 
+  private getGCMProperty(property: string){
+    // For some reason this isn't getting translated to the correct property in the java code.
+    return this.payload[`gcm.notification.${property}`] ?? this.payload[property];
```

Comment:
Should this be in the other order?

Reply:
The other one never exists and this is more or less what they use in java 🤷